lib/grinder_tools.dart

```diff
@@ -61,6 +61,7 @@ void defaultClean([GrinderContext context]) => delete(buildDir);
 /**
  * A utility class to run tests for your project.
  */
+@Deprecated('see `Test`')
```

Comment:
I thnk we should start using e.g. `'see [Test]'` instead pending the outcome of:

https://github.com/dart-lang/dartdoc/issues/500

but that can be a separate PR to update all of our deprecation messages.